### DISCUSSION TITLE
Mermaid 内で画像が表示されるようにしてほしい

### DISCUSSION THREAD

**Discussion by kyouheicf** [increments/qiita-discussions#547] [要望・アイデア・不具合]
例えば以下のような記法は Mermaid Live Editor では有効だが、Qiita では使えない
```
graph
Dir((<img src='https://iconscout.com/ms-icon-310x310.png' width='40' /><br>Search Icon Image))
bbb(<img src="data:image/svg+xml;charset=utf-8;base64,PHN2ZyBmaWxsPSIjMDA1MWMzIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxNiAxNiI+PHBhdGggZD0iTTcuMTI1IDIuNWwtLjY1IDFIM3Y5aDIuOWwtLjE3NSAxSDIuNUwyIDEzVjNsLjUtLjVoNC42MjV6bTMuMTUgMEgxMy41bC41LjV2MTBsLS41LjVIOC44NzVsLjY1LTFIMTN2LTloLTIuOWwuMTc1LTF6Ij48L3BhdGg+PHBhdGggZD0iTTcuMTUgOS41aC0yLjlsLS40MjUtLjc3NSA1LjItOCAuOS4zNzVMOC44NSA2LjVoMi45bC40MjUuNzc1LTUuMiA4LS45LS4zNzVMNy4xNSA5LjV6TTMuNzI1IDQuNTc1YS4zNS4zNSAwIDEwMC0uNy4zNS4zNSAwIDAwMCAuN3pNNC42NSA0LjU3NWEuMzUuMzUgMCAxMDAtLjcuMzUuMzUgMCAwMDAgLjd6TTUuNTc1IDQuNTc1YS4zNS4zNSAwIDEwMC0uNy4zNS4zNSAwIDAwMCAuN3oiPjwvcGF0aD48L3N2Zz4="/></br>Pages)```

**Comment by tanaka-asami415** [comment]
@kyouheicf 
リクエストありがとうございます！
こちら調査いたします！

**Reply by kyouheicf** [reply]
@tanaka-asami415 現時点で試してみたところ、画像は表示されるようになりました。
ただ `img` タグでのサイズ指定が効いていない :thinking: ので、 

<img width="318" alt="image" src="https://github.com/increments/qiita-discussions/assets/85217388/c6b7e1fa-ae8a-4d7d-9a8f-236be2a76779">

`themeCSS` を使って `%%{init: {'themeCSS': 'img {width: 48px; height:48px;}'}}%%` のように指定することでサイズ指定が適用されました。

**Comment by kimkim0814** [comment]
@kyouheicf
ご報告ありがとうございます。

バージョンアップにより10.3.1以降で画像が使えるようになりましたが、ご連絡できていませんでした🙏
[Mermaidのバージョンを10.2.4から10.9.0に変更しました](https://qiita.com/release-notes/2024-03-12)

`img`タグのサイズ指定の挙動については、他のスタイルの影響でそれが上書きされてしまっているのが原因かと思われます。
Mermaid公式のLive Editorと同様のためご容赦いただけると幸いです🙇

**Reply by kyouheicf** [reply]
@kimkim0814 ご確認ありがとうございます。承知しました🫡
